script.js

```diff
@@ -0,0 +1,521 @@
+      }
+    }
+  }
+}
+
+function changeLang() {
+  const rows = keyboard.children;
+  for (let row = 0; row < rows.length; row += 1) {
+    const keys = rows[row].children;
+    for (let key = 0; key < keys.length; key += 1) {
+      if (keys[key].querySelectorAll('.ru.on').length) {
+        keys[key].querySelectorAll('.en').forEach((item) => item.classList.add('on'));
+        keys[key].querySelectorAll('.ru').forEach((item) => item.classList.remove('on'));
+        window.localStorage.setItem('lang', 'en');
+      } else {
+        keys[key].querySelectorAll('.en').forEach((item) => item.classList.remove('on'));
+        keys[key].querySelectorAll('.ru').forEach((item) => item.classList.add('on'));
+        window.localStorage.setItem('lang', 'ru');
+      }
+    }
+  }
+}
+
+if (window.localStorage.getItem('lang') === 'en') {
+  changeLang();
+}
+
+// adding mousedown listeners for clicks on virtual keyboard
+document.addEventListener('mousedown', (e) => {
+  if (e.target.classList.contains('Tab')
+    || e.target.parentElement.parentElement.classList.contains('Tab')) {
+    addTab();
+  } else if (e.target.classList.contains('Enter')
+    || e.target.parentElement.classList.contains('Enter')) {
+    addEnter();
+  } else if (e.target.classList.contains('Backspace')
+    || e.target.parentElement.classList.contains('Backspace')) {
+    doBackspace();
+  } else if (e.target.classList.contains('Delete')
+    || e.target.parentElement.classList.contains('Delete')) {
+    doDelete();
+  } else if (e.target.classList.contains('ShiftLeft')
+    || e.target.parentElement.classList.contains('ShiftLeft')
+    || e.target.classList.contains('ShiftRight')
+    || e.target.parentElement.classList.contains('ShiftRight')) {
+    doShift();
+  } else if (e.target.classList.contains('ArrowLeft')
+    || e.target.parentElement.classList.contains('ArrowLeft')) {
+    doArrowLeft();
+  } else if (e.target.classList.contains('ArrowLeft')
+    || e.target.parentElement.classList.contains('ArrowLeft')) {
+    doArrowLeft();
+  } else if (e.target.classList.contains('ArrowRight')
+    || e.target.parentElement.classList.contains('ArrowRight')) {
+    doArrowRight();
+  } else if (e.target.classList.contains('ControlLeft')
+    || e.target.parentElement.classList.contains('ControlLeft')
+    || e.target.classList.contains('AltLeft')
+    || e.target.parentElement.classList.contains('AltLeft')
+    || e.target.classList.contains('AltRight')
+    || e.target.parentElement.classList.contains('AltRight')
+    || e.target.classList.contains('MetaLeft')
+    || e.target.parentElement.classList.contains('MetaLeft')
+    || e.target.classList.contains('MetaRight')
+    || e.target.parentElement.classList.contains('MetaRight')
+    || e.target.classList.contains('ArrowUp')
+    || e.target.parentElement.classList.contains('ArrowUp')
+    || e.target.classList.contains('ArrowDown')
+    || e.target.parentElement.classList.contains('ArrowDown')) {
+    // doNothing();
```

Comment:
If this part of code does nothing, than we should probably be able to remove it.